src/angular-drag-drop.js

```diff
@@ -1,6 +1,14 @@
-var Angular = require('angular');
+var Angular;
 
-module.exports = 'filearts.dragDrop';
+if (typeof angular === 'object') {
```

Comment:
1) This hacky change here, with the if and else, is to support the use of the library from apps that don't use webpack (they expect to have 'angular' already available globally, not to be imported using require).
Maybe there is another way to accomplish this configuring direclty webpack output bundle. 

2) The change in the angular module name is to fix the name itself, reading the README it seems that the angular module is called "angular-drag-drop" instead of "filearts.dragDrop".
Again, if the app using the library does not support the use of require it needs to specify directly the angular module name to be imported like this:

`angular.module('my-app', ['angular-drag-drop']);`

instead of:
```
var Angular = require('angular');
var mod = Angular.module('yourModule', [
  require('angular-drag-drop'),
]);

```